src/plone/restapi/tests/test_search.py

```diff
@@ -127,6 +128,17 @@ def test_search_on_context_constrains_query_by_path(self):
             set(result_paths(response.json())),
         )
 
+    def test_types_not_searched(self):
+        registry = getUtility(IRegistry)
+        search_settings = registry.forInterface(ISearchSchema, prefix="plone")
+        search_settings.types_not_searched = ('Folder',)
```

Comment:
For some reason, this configuration is changing when running the search.

Need help to figure out it.

Reply:
What do you mean?

Reply:
the test is failing, and when debugging it, realized that the `types_not_searched` configuration is missing before add the filter in the search ( `Folder` type is being listed)